This fixes the start and endfile specs to what actually is provided
--- a/gcc/config/xtensa/elf.h	2024-05-07 08:51:37.000000000 +0200
+++ b/gcc/config/xtensa/elf.h	2024-06-01 21:34:13.488232825 +0200
@@ -52,11 +52,11 @@ along with GCC; see the file COPYING3.
   %{mabi=call0:--abi-call0}"
 
 #undef LIB_SPEC
-#define LIB_SPEC "-lc -lsim -lc -lhandlers-sim -lhal"
+#define LIB_SPEC "-lc -lnosys -lc"
 
 #undef STARTFILE_SPEC
 #define STARTFILE_SPEC \
-  "crt1-sim%O%s crt0%O%s crti%O%s crtbegin%O%s _vectors%O%s"
+  "crt0%O%s crti%O%s crtbegin%O%s"
 
 #undef ENDFILE_SPEC
 #define ENDFILE_SPEC "crtend%O%s crtn%O%s"
